framework/elements/TextBox.js

```diff
@@ -0,0 +1,29 @@
+import BaseElement from "./BaseElement";
+
+class TextBox extends BaseElement{
+
+    constructor(locator, name){
+        super (locator, name);
+    }
+
+    async getText() {
+        const textBoxElement = await this.getElement();
+        return textBoxElement.getText();
```

Comment:
this  will not send you a text of an input. you need to use getValue method here